scripts/commands/wizard/quiz.js

```diff
@@ -0,0 +1,187 @@
+// USER FLOW:
+
+// DONE let's create a quiz
+// DONE do the lesson and tutorial exist?
+// DONE if not, bounce back to main menu
+// DONE confirm which tutorial
+// DONE confirm which lesson in an existing tutorial
+// DONE if no mult choice lesson there, bounce back to main menu
+// TODO confirm overwrite existing quiz if there's real data
+// DONE prompt for question
+// DONE prompt for correct answer (text, success message)
+// DONE prompt repetitively for 2-4 wrong answers (text, failure message)
+// DONE randomly sort order of wrong answers
+// DONE store data to js file (remember to deal with quotes correctly)
+// DONE log randomized questions?
+// DONE show link to preview
+// DONE show which file to go to to edit text
+// DONE create another quiz OR exit
+
+const log = require('npmlog')
+const inquirer = require('inquirer')
+const api = require('../../../src/api')
+
+const {
+  selectTutorial,
+  promptFilesReady,
+  selectLesson,
+  logList,
+  validateStringPresent,
+  promptRepeat,
+  logPreview,
+  logCreateLater
+} = require('./utils.js')
+
+// *** QUIZ CREATION ***
+
+async function createQuiz (tutorial, lesson, { createLesson, createTutorial, createResource }) {
+  log.info("Now it's time to write the question for your multiple-choice quiz and provide answer choices, with positive or negative feedback for each. Wrong answers, and the feedback associated with them, are a great way to address common misconceptions about the topic. Be sure to make your feedback as helpful as possible, to guide the learner to the right choice. You'll need to create 1 correct answer and 2-3 incorrect answers. (I'll take care of randomizing their order later.")
+  const responses1 = await inquirer.prompt([
```

Comment:
we can rename this to `responses` and on the bellow variable `responses2` we can simply reuse the initial one.